__tests__/defaultProvider.test.ts

```diff
@@ -81,6 +81,14 @@ describe('defaultProvider', () => {
         const block = await testProvider.getBlock('latest');
         return expect(block).toHaveProperty('block_number');
       });
+
+      test('getStateUpdate()', async () => {
```

Comment:
Adjust and expand the test to use the `exampleBlock` values from the `beforeAll` setup step to avoid false positives.